pandas/core/arrays/numpy_.py

```diff
@@ -137,8 +138,8 @@ def __init__(self, values, copy=False):
         if copy:
             values = values.copy()
 
-        self._ndarray = values
-        self._dtype = PandasDtype(values.dtype)
+        self._ndarray: np.ndarray = values
+        self._dtype: ExtensionDtype = PandasDtype(values.dtype)
```

Comment:
is PandasDtype valid here?  seems more specific

Reply:
I'll remove for now. mypy may no longer require this type annotation.